prow/config/external_plugins_config.yaml

```
@@ -1875,3 +1874,29 @@ ti-community-issue-triage:
     linked_issue_needs_triage_label: "do-not-merge/needs-triage-completed"
     need_cherry_pick_label_prefix: "needs-cherry-pick-release-"
     status_target_url: "https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/cherrypick-a-pr.html#pass-triage-complete-check"
+  - repos:
+      - tikv/tikv
+    maintain_versions:
+      - "4.0"
+      - "5.0"
+      - "5.1"
+      - "5.2"
+      - "5.3"
+      - "5.4"
+      - "6.0"
+      - "6.1"
+      - "6.2"
+      - "6.3"
+    wontfix_versions:
+      - "4.0"
+      - "5.0"
+      - "5.1"
+      - "5.2"
+      - "6.0"
+      - "6.2"
+      - "6.3"
+    affects_label_prefix: "affects-"
+    may_affects_label_prefix: "may-affects-"
+    linked_issue_needs_triage_label: "do-not-merge/needs-triage-completed"
+    need_cherry_pick_label_prefix: "needs-cherry-pick-"
```

```suggestion
    need_cherry_pick_label_prefix: "needs-cherry-pick-release-"
```

tikv/tikv is using cherry-pick function in TiChiBot